### PR TITLE
Updated documentation for updating snapshots

### PR DESCRIPTION
**Pull request by WayneFerrao**
Updated documentation for snapshot tests to provide guidance if the update script doesn't work as expected.

### CODE REVIEW

**Review by andre4i — APPROVED** [review]
I would also add some instructions about how to test a change in the `FluidFrameworkTestData` in the CI before committing to `main` (pointing the submodule to the new branch in `FluidFrameworkTestData`).

File: packages/test/snapshots/README.md [hunk 1]
```diff
@@ -117,6 +118,10 @@ Whenever the snapshot format changes in the runtime, the existing snapshots in t
   npm run test:update
   ```
   This will save the current snapshots in the `src_snapshots\<current_snapshots_version>` directory. It will then update the current snapshots under the `current_snapshots` directory to the new format.
+- However, there is a chance that not all the snapshots will be updated. You will see this as a failure in the Azure build, which will indicate which files contain snapshots that were not updated. The failures might have to do with the same property/value and is likely present in all the files within that subdirectory. To solve this, do the following:
```
Comment (agarwal-navin) [inline packages/test/snapshots/README.md:121]: Can you give an example for this failure? I am curious what is leading this to happen. Not sure I like the step of manually updating the snapshots as that can lead to a lot of issues later.

File: packages/test/snapshots/README.md [hunk 2]
```diff
@@ -37,10 +37,11 @@ For submitting changes to the snapshot test content, follow these steps:
     - [Adding new snapshots](#Adding-new-snapshots-to-the-repo) - If you are adding new snapshots.
     - [Updating existing snapshots](#Updating-existing-snapshots-in-the-repo) - If you updating existing snapshots.
   - Create a commit and push the new branch.
-  - Create a PR against `main` branch in the [FluidFrameworkTestData](https://github.com/microsoft/FluidFrameworkTestData) repo. Get the changes approved and merge the PR.
-  - Checkout `main` branch and run `git pull` to ensure it has your new changes.
+  - Create a PR against `main` branch in the [FluidFrameworkTestData](https://github.com/microsoft/FluidFrameworkTestData) repo. Ensure the build passes, get the changes approved and merge the PR.
+  - Checkout `main` branch in the content/ folder and run `git pull` to ensure it has your new changes.
```
Comment (agarwal-navin) [inline packages/test/snapshots/README.md:41]:
```suggestion
  - Checkout `main` branch of FluidFrameworkTestData repo, i.e., in the `content/` folder and run `git pull` to ensure it has your new changes.
```

File: packages/test/snapshots/README.md (hunk 1 above)
Comment (WayneFerrao) [reply packages/test/snapshots/README.md:121]: So my code change was to remove all instances of summaryCount. After running the test:update script, there were still a few instances where summaryCount was expected. But you're right, manually updating the snapshots is definitely not ideal. 
https://dev.azure.com/fluidframework/public/_build/results?buildId=80731&view=ms.vss-test-web.build-test-results-tab&runId=1765448&resultId=100003&paneView=debug

**Pull request merged**